EnglishFilter/sections/antiadblock.txt

```diff
@@ -568,6 +568,10 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/77887
+insidertracking.com#%#//scriptlet("prevent-setTimeout", "no_ad", "15000")
+!+ PLATFORM(ios, ext_android_cb)
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=insidertracking.com
```

Comment:
Move it to the `content_blocker.txt` without hint.

Reply:
Okay.